sumomongodbatlascollector/api.py

```diff
@@ -38,9 +38,11 @@ def get_window(self, last_time_epoch):
 
         return start_time_epoch, end_time_epoch
 
+	@staticmethod
     def _get_cluster_name(self, full_name_with_cluster):
         return full_name_with_cluster.split("-shard")[0]
 
+	@staticmethod
```

Comment:
indentation seems incorrect, did we run flake 8 over this ?

Reply:
Have fixed the linting and formatting